Makefile

```diff
@@ -1,9 +1,21 @@
-make test_backend:
-	docker-compose run --rm backend python3 manage.py test && pylint apps
+test_backend:
+	docker-compose run --rm backend python3 manage.py test
 
-make test_frontend:
+test_frontend:
 	docker-compose run --rm frontend node_modules/.bin/ng test --single-run
 
-make test_all:
-	make test_backend
-	make test_frontend
+test_all: test_backend test_frontend
+
+populate_database:
+	docker-compose run --rm backend python3 manage.py populate_database
+
+# Beware - Following commands delete ALL docker images and containers!
+docker_remove_containers:
+	docker rm $$(docker ps -qa --filter name='volontulo_*')
+
+docker_remove_images:
+	docker rmi $$(docker images -qa --filter reference='*/volontulo*')
+	# Unfortunately, we do not have a dedicated volontulo postgres image I guess
+	docker rmi $$(docker images -qa --filter reference='postgres')
```

Comment:
No need to remove postgres image - it can be shared between other projects and doesn't change often.

Reply:
+1